config-solaris.md

```diff
@@ -73,31 +73,31 @@ cappedMemory is mapped to capped-memory in zonecfg(8) man page.
 anet is specified as an array that is used to setup networking for Solaris application containers.
 The anet resource represents the automatic creation of a network resource for an application container.
 The zones administration daemon, zoneadmd, is the primary process for managing the container's virtual platform.
-One of the daemons is responsibilities is creation and teardown of the networks for the container.
-For more information on the daemon check the zoneadmd(1M) man page.
+One of the daemon's responsibilities is creation and teardown of the networks for the container.
+For more information on the daemon see the [zoneadmd(1M)][zoneadmd.1m] man page.
 When such a container is started, a temporary VNIC(Virtual NIC) is automatically created for the container.
 The VNIC is deleted when the container is torn down.
 The following properties can be used to setup automatic networks.
-For additional information on properties check zonecfg(8) man page for the respective release of Solaris.
+For additional information on properties, check the [zonecfg(1M)][zonecfg.1m] man page for the respective release of Solaris.
 
 * **`linkname`** *(string, OPTIONAL)* Specify a name for the automatically created VNIC datalink.
 * **`lowerLink`** *(string, OPTIONAL)* Specify the link over which the VNIC will be created.
-Mapped to lower-link in the zonecfg(8) man page.
+Mapped to 'lower-link' in the [zonecfg(1M)][zonecfg.1m] man page.
```

Comment:
I'm a fan of marking these up, but I'd rather use backticks (`lower-link`) like we do in most other parts of the spec.